### DISCUSSION TITLE
Lazer Round Robin, please let host delete maps.

### DISCUSSION THREAD

**Discussion by Veldora2k** [ppy/osu#15947] [Ideas] [answered]
Round robin when you let everybody pick 3 maps.

Most of the times you want to play at specific star level, for example 4.7-5.7*. Even if you type this in room name, there are some kind of players....(mostly russians) that put something different (for example 1 star map or 7 star map). Sometimes people just misclicked and pick wrong difficulty level or mod.

->Please let host delete map in poll, so some kind of mistakes can be simply undone.
Now you can only start solo, stay on map for 2-3 sec and left to skip specific map.

**Comment by bdach** [comment, marked as answer]
known missing feature. see https://github.com/ppy/osu/pull/15640

> * Item owners should be able to delete their own items in free-for-all and fair-rotate modes.
> * Item owners should be able to edit their own items in free-for-all and fair-rotate modes.
> * The host should be able to delete/change items in all modes.
